utils.go

```diff
@@ -227,13 +227,28 @@ var supportedHeaders = []string{
 	// Add more supported headers here.
 }
 
+var storageClassHeader = []string{
```

Comment:
You don't need this to be a slice, this can be simply kept single string and use amzStorageClass?